docs/chain_documentation/workflow/osm/data_preparation.md

```diff
@@ -1,6 +1,68 @@
 # OSM  - Extraction and transformation to GIS layers
 
+Figure 1 shows the sequence of actions that are performed to retrieve OSM data and transform them into GIS layers.
+
+![osm_extraction](../../../resources/images/chain_documentation/osm_extract_gis_layers.png)
+
+Figure 1. OSM data flow transformation
+
+The sequence starts either by specifying a place name or by specifying an OSM bbox.
```

Comment:
Why are you talking about an OSM bbox. It's a bbox ?!
We say place name or placename ?

Reply:
osm bbox is a specific bbox

Reply:
all system do not use the same bbox

Reply:
You mean in terms of coordinate system ? Or the way we list the coordinates ?

Reply:
the order of the lat, lon are not the same

Reply:
placename is the identifier in code.